CHANGELOG.rst

```diff
@@ -18,6 +18,21 @@ The format is based on `Keep a Changelog`_.
 `UNRELEASED`_
 =============
 
+Added
+-----
+
+Changed
+-------
+
+Removed
+-------
+
+Fixed
+-----
+
+`0.5.5`_ - 2018-07-02
```

Comment:
Can you update the links at the bottom of the file, updating the url for `UNRELEASED` and adding an entry for 0.5.5?

```
.. _UNRELEASED: https://github.com/QISKit/qiskit-core/compare/0.5.5...HEAD
.. _0.5.5: https://github.com/QISKit/qiskit-core/compare/0.5.4...0.5.5
```